cli/admin/src/test/java/io/pravega/cli/admin/dataRecovery/DataRecoveryTest.java

```diff
@@ -1714,6 +1718,29 @@ public void testLTSRecoveryCommand() throws Exception {
         Assert.assertNotNull(RecoverFromStorageCommand.descriptor());
     }
 
+    @Test
+    public void testLTSRecoveryNullEntry() throws Exception {
+        STATE.set(new AdminCommandState());
+        Properties pravegaProperties = new Properties();
+        pravegaProperties.setProperty("pravegaservice.container.count", "1");
+        STATE.get().getConfigBuilder().include(pravegaProperties);
+
+        CommandArgs args = new CommandArgs(List.of("/tmp/metadata", "all"), STATE.get());
+        RecoverFromStorageCommand command = new RecoverFromStorageCommand(args);
+        @Cleanup
+        DebugStreamSegmentContainer container = Mockito.mock(DebugStreamSegmentContainer.class);
+        ContainerTableExtension extension = Mockito.mock(ContainerTableExtension.class);
+        List<TableEntry> entries = new ArrayList<>();
+        entries.add(null);
+        CompletableFuture<List<TableEntry>> cf = CompletableFuture.supplyAsync(() -> entries);
+        Mockito.doReturn( cf ).when(extension).get(Mockito.any(), Mockito.any(), Mockito.any());
+        Mockito.doReturn(extension).when(container).getExtension(Mockito.any());
+        RecoverFromStorageCommand.ChunkValidator chunkValidator = Mockito.spy(command.new ChunkValidator(container));
```

Comment:
nit: Can we import `ChunkValidator`?

Reply:
Same for `spy`, `mock` and `assertTrue` as well.
eg. - `org.mockito.Mockito.spy`

Reply:
Doing so is affecting all Mockito imports in the file in IDE showing a huge diff. So just keeping it the way it is in this Test file.

Reply:
hand made these changes.